doc/index.rst

```diff
@@ -22,4 +22,5 @@ For stability we recommend cloning or checking out a tagged release.
    :maxdepth: 2
 
    tutorial
+   examples
```

Comment:
I believe you are missing the `example.rst` file that calls the `ex1.py` script via `literalinclude`

Reply:
Forgot to commit that one. Should be there now.